libs/SFMLResource/CMakeLists.txt

```diff
@@ -4,8 +4,24 @@ set(TARGET ${PROJECT_NAME}_SFMLResource)
 
 set(SRC
     ${INC_SFML_RESOURCE_ROOT}/RenderWindowResource.hpp
+    ${INC_SFML_RESOURCE_ROOT}/GraphicsFontResource.hpp
+    ${INC_SFML_RESOURCE_ROOT}/GraphicsTextureResource.hpp
 )
 
-add_library(${TARGET} INTERFACE ${SRC})
+add_library(${TARGET} SHARED ${SRC})
```

Comment:
Please put it in archive while the SHARED refacto has not been made

Reply:
BTW, you should changed it to an Interface cause there's no source file

Reply:
There are some source files like GraphicsFontResource.cpp or GraphicsTextureResource.cpp